### ISSUE TITLE
Failed to start the application with docker-compose,  receive exception in server and work service

### ISSUE THREAD

**Issue by ToughB** [inventree/InvenTree#7370]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

the service : server keeps restarting. The service:worker is running. But when looking at the logs, it can be seen that an exception occurs within each of the two services. I have looked at the following logs.

It seems as if missing database entries lead to the expetions

### Steps to Reproduce

1. Cloning repo Branch:master
2. docker-compose up within the contrib/container directory

### Expected behaviour

nothing special

### Deployment Method

- [X] Docker
- [ ] Package
- [ ] Bare metal
- [ ] Other - added info in Steps to Reproduce

### Version Information

lates , stable 

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

### Relevant log output

```shell
### service : server


| Traceback (most recent call last):
inventree-server    |   File "/root/.local/bin/gunicorn", line 8, in <module>
inventree-server    |     sys.exit(run())
inventree-server    |              ^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
inventree-server    |     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]", prog=prog).run()
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/base.py", line 236, in run
inventree-server    |     super().run()
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/base.py", line 72, in run
inventree-server    |     Arbiter(self).run()
inventree-server    |     ^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 58, in __init__
inventree-server    |     self.setup(app)
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 118, in setup
inventree-server    |     self.app.wsgi()
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
inventree-server    |     self.callable = self.load()
inventree-server    |                     ^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
inventree-server    |     return self.load_wsgiapp()
inventree-server    |            ^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
inventree-server    |     return util.import_app(self.app_uri)
inventree-server    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
inventree-server    |     mod = importlib.import_module(module)
inventree-server    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
inventree-server    |     return _bootstrap._gcd_import(name[level:], package, level)
inventree-server    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
inventree-server    |   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
inventree-server    |   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
inventree-server    |   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
inventree-server    |   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
inventree-server    |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
inventree-server    |   File "/home/inventree/src/backend/InvenTree/InvenTree/wsgi.py", line 17, in <module>
inventree-server    |     application = get_wsgi_application()  # pragma: no cover
inventree-server    |                   ^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
inventree-server    |     django.setup(set_prefix=False)
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/__init__.py", line 24, in setup
inventree-server    |     apps.populate(settings.INSTALLED_APPS)
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/apps/registry.py", line 124, in populate
inventree-server    |     app_config.ready()
inventree-server    |   File "/home/inventree/src/backend/InvenTree/InvenTree/apps.py", line 54, in ready
inventree-server    |     self.start_background_tasks()
inventree-server    |   File "/home/inventree/src/backend/InvenTree/InvenTree/apps.py", line 99, in start_background_tasks
inventree-server    |     for existing_task in Schedule.objects.all():
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/models/query.py", line 398, in __iter__
inventree-server    |     self._fetch_all()
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/models/query.py", line 1881, in _fetch_all
inventree-server    |     self._result_cache = list(self._iterable_class(self))
inventree-server    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
inventree-server    |     results = compiler.execute_sql(
inventree-server    |               ^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
inventree-server    |     cursor.execute(sql, params)
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
inventree-server    |     return self._execute_with_wrappers(
inventree-server    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
inventree-server    |     return executor(sql, params, many, context)
inventree-server    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
inventree-server    |     with self.db.wrap_database_errors:
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
inventree-server    |     raise dj_exc_value.with_traceback(traceback) from exc_value
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
inventree-server    |     return self.cursor.execute(sql, params)
inventree-server    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
inventree-server    |   File "/root/.local/lib/python3.11/site-packages/psycopg/cursor.py", line 732, in execute
inventree-server    |     raise ex.with_traceback(None)
inventree-server    | django.db.utils.ProgrammingError: relation "django_q_schedule" does not exist
inventree-server    | LINE 1: ...", "django_q_schedule"."intended_date_kwarg" FROM "django_q_...
inventree-server    |                                                              ^
```

### service : inventree-db Logs

```
inventree-db        | 2024-05-29 09:39:50.669 UTC [34] ERROR:  relation "common_inventreesetting" does not exist at character 112
inventree-db        | 2024-05-29 09:39:50.669 UTC [34] STATEMENT:  SELECT "common_inventreesetting"."id", "common_inventreesetting"."value", "common_inventreesetting"."key" FROM "common_inventreesetting" WHERE UPPER("common_inventreesetting"."key"::text) = UPPER('REPORT_DEFAULT_PAGE_SIZE') ORDER BY "common_inventreesetting"."id" ASC LIMIT 1
inventree-db        | 2024-05-29 09:39:50.670 UTC [34] ERROR:  relation "common_inventreesetting" does not exist at character 22
inventree-db        | 2024-05-29 09:39:50.670 UTC [34] STATEMENT:  SELECT 1 AS "a" FROM "common_inventreesetting" WHERE "common_inventreesetting"."key" = 'REPORT_DEFAULT_PAGE_SIZE' LIMIT 1
inventree-db        | 2024-05-29 09:39:50.671 UTC [34] ERROR:  relation "common_inventreesetting" does not exist at character 112
inventree-db        | 2024-05-29 09:39:50.671 UTC [34] STATEMENT:  SELECT "common_inventreesetting"."id", "common_inventreesetting"."value", "common_inventreesetting"."key" FROM "common_inventreesetting" WHERE UPPER("common_inventreesetting"."key"::text) = UPPER('_PLUGIN_REGISTRY_HASH') ORDER BY "common_inventreesetting"."id" ASC LIMIT 1
inventree-db        | 2024-05-29 09:39:50.672 UTC [34] ERROR:  relation "report_billofmaterialsreport" does not exist at character 13
inventree-db        | 2024-05-29 09:39:50.672 UTC [34] STATEMENT:  INSERT INTO "report_billofmaterialsreport" ("metadata", "name", "template", "description", "revision", "page_size", "landscape", "filename_pattern", "enabled", "filters") VALUES ('{}'::jsonb, 'Bill of Materials', 'report/inventree/bom/inventree_bill_of_materials_report.html', 'Bill of Materials report', 2, 'A4', false, 'report.pdf', true, '') RETURNING "report_billofmaterialsreport"."id"
```
```

**Comment by matmair** [comment]
Reading the docs on starting up docker might help: https://docs.inventree.org/en/stable/start/docker_install/#initial-database-setup

**Comment by ToughB** [comment]
Thanks for the hint. Although I skimmed through the documentation, I didn't notice this point. Perhaps these necessary steps would be worth mentioning as a comment in the docker-compose file ?

**Comment by SchrodingersGat** [comment]
@ToughB can you please link to the docs that you did follow?

**Comment by ToughB** [comment]
@SchrodingersGat  I originally landed on the right page. But after I reached the point : Point Before You Start...my thinking error came into play. I thought I had to click on the STABLE link to get to the documentation for the STABLE version in conjunction with Docker. I didn't find anything there regarding additional commands that should be executed before the docker-compose up -d. Sorry for my hasty action. It was not my intention to criticize anything. Instead, I want to thank you for the help and for the application. I like it very much so far.
